tools/autograd/templates/Functions.cpp

```diff
@@ -146,8 +146,19 @@ Tensor pow_backward_self(Tensor grad, const Tensor & self, const Tensor & expone
   return at::where(exponent == 0.0, at::zeros({}, grad.options()), grad * exponent * self.pow(exponent - 1));
 }
 
-Tensor pow_backward_exponent(Tensor grad, const Tensor & self, Tensor result) {
-  return at::where(self == 0, at::zeros({}, grad.options()), grad * result * self.log());
+// Caveats:
+// We define d(a^b)/db at a = 0 and b < 0 to be -inf. This is due to
+// * d(a^b)/db -> -inf for a fixed b as a -> +0
+// * convenience of implementation: no need to have a second at::where statement
```

Comment:
I don't think this point is needed.
The one above where we define it by continuity is all we need.

Reply:
Done